### DISCUSSION TITLE
Sunshine API

### DISCUSSION THREAD

**Discussion by cyrebre** [mozts2005/ZendeskApi_v2#499] [Features]
Hi,
Are there any plans to support Sunshine API ? 
https://developer.zendesk.com/rest_api/docs/sunshine/introduction#sunshine

Many thanks for your great API.
Cyril

**Comment by mozts2005** [comment]
I am planning to add it to the rewrite of this project. but as of now this is the only request for Sunshine API. I would be willing to except a community contribution or if the community wanted to fund the effort.

**Comment by cyrebre** [comment]
I think I can help you with this, a bit later this year...
________________________________
